code/online_challenges/src/codeforces/round_615/div3_problem_B.cpp

```diff
@@ -0,0 +1,83 @@
+#include<bits/stdc++.h>
+using namespace std;
+#define ll long long
+#define ld long double
+#define ull unsigned long long
+#define str string
+#define pb push_back
+#define fr(i,n) for(ll i=0; i<n; i++)
```

Comment:
@rudrakshi99 The code will be more readable if you remove macro and templates and implemented it fully in code.